README.md

```diff
@@ -1,6 +1,14 @@
 # AI Multimodal Public Demos
 
-The demos in this repostiory rely solely on public data sets.
+This repository is part of [AIMM project](https://github.com/AI-multimodal): a joint and collaborative effort between scientist from [Brookhaven National Laboratory (BNL)](https://www.bnl.gov/) and [Lawrence Berkeley National Laboratory (LBL)](https://www.lbl.gov/)
+Current contributors working in this project are: Dylan McReynolds and Jospeh Kleinhenz from LBL, and Dan Allan and Juan Marulanda from BNL.
```

Comment:
We should include all coauthors on the grant and all the post-docs hired through the grant.

Reply:
Do we have a list of all the post-docs in all the participating facilities?